tests/test_artifacts/context_caller_contract/Forc.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "context_caller_contract"
+
+[dependencies]
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
```

Comment:
Remove `core` unless a special version of it is used (which is not the case here).